shared/common/build/tools/update_podspec_headers.sh

```diff
@@ -30,6 +30,8 @@
 # set -x
 set -e
 
+VERSION=4.0.1 # XXX: update that version for every release
```

Comment:
We could start maintaining a "version number" file at the root of the repo, which could help in this scenario.